OnlinerTask/OnlinerTask/Controllers/ProductController.cs

```diff
@@ -8,106 +8,36 @@
 using System.Runtime.Serialization.Json;
 using System.Threading.Tasks;
 using System.Web.Http;
+using OnlinerTask.Models;
+using OnlinerTask.Services;
 
 namespace OnlinerTask.Controllers
 {
     [Authorize]
     public class ProductController : ApiController
     {
-        // GET: api/Product
+
         public string Get()
         {
             return "value";
         }
-
-        // POST: api/Product
+        
         public async Task<List<ProductModel>> Post(Responce responce)
         {
             if (responce == null || String.IsNullOrEmpty(responce.SearchString))
                 return null;
-            HttpWebRequest request = Service.OnlinerRequest(responce.SearchString);
+            HttpWebRequest request = SearchService.OnlinerRequest(responce.SearchString);
```

Comment:
Is SearchService singleton? If so - remove singleton instance and set up injection as constructor parameter.